docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst

```diff
@@ -26,16 +26,17 @@ Prerequisites
 ^^^^^^^^^^^^^
 
 To run the demo, you will need access
-to a Daml Enterprise release, the
+to a Daml Enterprise with drivers release, the
```

Comment:
Unless the "Daml Enterprise with drivers" is a defined product line I would say "Daml Enterprise release with drivers"

Reply:
ammended